src/hooks/use-authentication.ts

```diff
@@ -21,8 +37,31 @@ const signInWrapper = (provider: ValidAuthProviderId = DEFAULT_PROVIDER_ID) => {
  * https://next-auth.js.org/getting-started/client#signout
  */
 const signOutWrapper = (options: SignOutParams = {}) => {
-	const { callbackUrl = '/', redirect = true } = options
-	return signOut({ callbackUrl, redirect })
+	const { callbackUrl = '/', redirect = true, ...restOptions } = options
```

Comment:
I think we'll want to be able to pass a callback URL here also, similar to the sign-in. Not necessary to add here though, but a note might be nice.

Reply:
Would those `restOptions` override this `callbackUrl` if it was passed?

Reply:
So both `callbackUrl` and `redirect` can be overwritten by providing them in the `options` object, but they are not overwritten by `restOptions`. I've added the extra destructuring line of code in order [to provide default values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value) for both `callbackUrl` and `redirect`, then pulling `restOptions` so they can be put back into the object sent to `signOut` untouched.